include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -226,6 +270,19 @@ SECTIONS
 	} GROUP_DATA_LINK_IN(RAMABLE_REGION, RAMABLE_REGION)
 #endif
 
+#ifdef CONFIG_SOC_SERIES_PSOC62
```

Comment:
looks like we need a better way to add soc specific sections into this general .ld file. Adding SoC specific section will make this file explode with ifdefs.

Reply:
Added something similar to the custom-*.ld for SoC.